src/main/js/config/BranchList.tsx

```diff
@@ -82,17 +81,15 @@ class BranchList extends React.Component<Props, State> {
             {branches.map(branch => (
               <tr>
                 <td>{branch}</td>
-                <VCenteredTd className="is-darker">
-                  <a
-                    className="level-item"
+                <WidthTd className="has-text-centered">
+                  <span
```

Comment:
Why not a styled button ? Wouldn't that be better semantically ?

Reply:
The associated ticket mentions a larger click area, but I still can only click the small icon.